src/pimp_mixer.c

```diff
@@ -3,6 +3,10 @@
  * For conditions of distribution and use, see copyright notice in LICENSE.TXT
  */
 
+#ifdef TARGET_GBA
+#include <gba.h>
+#endif
```

Comment:
It's unclear why you want to change this...

Reply:
I was compiling it with LTO, which made the `__thumb__` macro be mistakenly (not) defined because the actual code generated happened in a later step.

I do understand the argument for not wanting to rely on libgba, but I'm not sure how to remove this dependency *and* enable LTO (except for adding this `Div` function separately somewhere). I simply took this because it's the code I used for [Uitslaganticipatieangst 2](https://www.pouet.net/prod.php?which=84533).

Reply:
Right. We don't use LTO for pipmobile, so this sounds like user-error ;)

I think you should carry this patch in your own fork for now. I don't think compiling pimpmobile with LTO is a worthwhile goal right now.

Reply:
Alright, I'll roll it back.